### DISCUSSION TITLE
Class '' not found

### DISCUSSION THREAD

**Discussion by jbello94** [spatie/laravel-permission#1714] [Q&A] [answered]
After following the steps to install the package in Lumen (https://spatie.be/docs/laravel-permission/v4/installation-lumen), when I run the command '`php artisan migrate`' I get the error   **Class '' not found**

**Comment by drbyte** [comment, marked as answer]
You must have missed a step.

I just followed all the steps as listed in the doc and the migration works fine without error.
a. Create the new lumen project with the composer command listed in the [Lumen Installation docs](https://lumen.laravel.com/docs/master)
b. Create a MariaDB database, and set its name and credentials in the `.env`
c. Follow all the instructions at https://spatie.be/docs/laravel-permission/v4/installation-lumen

The output of `php artisan migrate` is as expected:
```
Migration table created successfully.
Migrating: 2018_01_01_000000_create_permission_tables
Migrated:  2018_01_01_000000_create_permission_tables (416.60ms)
```
(Yes the 2018 date comes directly from using the copy command in the https://spatie.be/docs/laravel-permission/v4/installation-lumen doc. It is exactly as expected. You can of course use your own date by renaming the migration file after copying it and before running the migration.)
